LoopKit/Extensions/HKQuery.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  HKQuery.swift
+//  LoopKit
+//
+//  Created by Rick Pasetto on 6/22/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import HealthKit
+
+public var excludeHealthKitDataFromOtherApps = false
```

Comment:
This should be an argument to the stores, not a global var. I'm all for convenience, but this crosses the line into code debt, imo. The stores are designed as separately instantiable objects, and configuration of them should be done via their constructor or methods/variables exposed on on them.

Reply:
Good suggestion. 👍   I was not a huge fan of this anyway.